tests/functional/pv/pv_services/test_cephfs_fsync_consistency.py

```diff
@@ -0,0 +1,91 @@
+import logging
+
+from ocs_ci.framework.testlib import ManageTest, tier2, bugzilla, green_squad
+from ocs_ci.helpers import helpers
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.node import get_worker_nodes
+
+
+logger = logging.getLogger(__name__)
+
+
+@tier2
+@bugzilla("2212310")
+@green_squad
+class TestCephfsFsyncConsistency(ManageTest):
+    """
+    Ensuring File Completeness Post-fsync with Shared CephFS Volume
+
+    """
+
+    def test_cephfs_fsync_consistency(self, teardown_project_factory):
+        """
+        Procedure:
+            1.Set Up Shared Storage: Create a PVC with sc=cephfs and mode=RWX.
+            2.Deploy Client Pod on Worker Node A, attaching the shared PVC.
+            3.Deploy Server Pod on Worker Node B, attaching the same PVC.
+            4.Client Pod write a file to the shared PVC, then call fsync to ensure data is flushed to disk.
+            5.Server Pod Read the file
+        """
+        project_name = "cephfs-rwx-ns"
+        pod_name = "cephfs-rwx-pod"
+        project_obj = helpers.create_project(project_name=project_name)
```

Comment:
any reason for not using project_factory_fixture ?

Reply:
I don't see anyone using `project_factory_fixture `in the test file.